languages/tree-sitter-stack-graphs-java/test/method_in_extended_class.java

```diff
@@ -0,0 +1,25 @@
+class TestClass extends SecondTestClass {
+                        // ^ defined: 16
+  public static void main(String[] args){
+    foo();
+ // ^ defined: 10
+    bar();
+//  ^ defined: 17
+  }
+
+  public void foo() {
+    super.bar();
+        // ^ defined: 17
+  }
+}
+
```

Comment:
I'd suggest to factor out the use of `super` into a separate test. I think the easiest way to have `super` is introducing a `super` variable declaration, linking that to the superclass reference, and treating `super` similar to other variable references. This is enough involved parts to test is separately from the normal superclass access, I'd say.

Reply:
That works for me. I imagine it would be useful to have separate tests for super variables and methods. As the code stands right now, super variable references do not work yet. I'll add it in as part of this PR :)